### DISCUSSION TITLE
Sensor will not warm up

### DISCUSSION THREAD

**Discussion by NIkolovafa** [NightscoutFoundation/xDrip#2936] [Q&A]
Hi all,

I have been trying to set up my son's sensor. We are using Dexcom one sensor with Xdrip on Samsung A13 device. All was working fine, until I had to change the transmitter. The new one starts with letter C. The Xdrip and the android are updated with the latest version. The transmitter is connected. However, when I put the sensor code it starts warming up and stops after 5-10 min with a message: Now star your sensor or check setting. I tried with two different sensors. Checked all settings, it will not warm. 
The only thing that I see in the logs is: Sensor Start failed: Clock not synchronized or another error. 

Any help will be much appreciated. Thanks.

**Comment by Navid200** [comment]
Your system status page shows March 28, 2023.
That's not the latest.
Just in case you are interested to have a look later, I have explained how xDrip update works here:  https://navid200.github.io/xDrip/docs/Updates.html

But, for now, please start by updating to this version to begin with:
https://github.com/NightscoutFoundation/xDrip/releases/tag/2023.05.03

Never ever snap the transmitter into a new sensor right after you peel it off with the previous sensor.
After the sensor is stopped and confirmed on the status page, you should remove the transmitter and keep it out for at least 10 minutes before snapping it into a new sensor.

You can try a restart right now using the same sensor to avoid wasting yet another sensor.
https://navid200.github.io/xDrip/docs/Restart-G6-sensor.html

**Reply by NIkolovafa** [reply]
Hi,

Thanks for your prompt response. 

The screenshot is before I updated. After the update I get the same results.

All was working fine, the transmitter was reaching the end of life, and with the new sensor, I started a new transmitter, this is when the issue stared. 
It starts warming up and stops itself after few minutes. After that I put the code again, it starts warming up and stops again. After many tries, I removed the sensor. Few days later tried with a new sensor, still the same. 
There isn't an option stop sensor as it just stops on its own.
I checked all the settings many times. Currently trying to hard reset. 

Thanks.

**Reply by Navid200** [reply]
After it fails, you cannot just start and expect it to work.
You cannot hard reset that.  It does not work.

Follow the instructions I provided for restart.  Don't skip steps.

**Reply by Navid200** [reply]
There are some xDrip messages that are temporary.  You should not react when you see a message.  Wait for the transmitter to respond.
Everything only happens once every 5 minutes.
If you see a failure message, give it time.  Does it still show the message after 5-10 minutes?

**Reply by NIkolovafa** [reply]
It does.

**Reply by Navid200** [reply]
It does what?

**Reply by NIkolovafa** [reply]
Basically there is no error message showing on the screen. It only shows stopped sensor. And It stays like that, until I put the code again.
![16889997620647644330598821672468](https://github.com/NightscoutFoundation/xDrip/assets/92533218/ad2a079a-cf35-48f0-a255-7833c10ca66c)

**Reply by Navid200** [reply]
Please don't look at the main screen.  Everything that is important is on the status page.
Do you mind posting the latest status page?
Would you pleas not post an outdated screenshot.  I want to help you.

**Reply by NIkolovafa** [reply]
Thanks for that. I really appreciate your time. This is what the latest status page shows:
![1689001054739203485706627930046](https://github.com/NightscoutFoundation/xDrip/assets/92533218/80803a10-6c63-47eb-ac73-b8b69e1e89ce)

**Reply by Navid200** [reply]
Thanks.  That all looks very good.

If you follow the restart instructions to the letter, it should function properly.  here are the instructions again:
https://navid200.github.io/xDrip/docs/Restart-G6-sensor.html

**Reply by NIkolovafa** [reply]
Unfortunately, that didn't work out 😕
![IMG_20230710_191646](https://github.com/NightscoutFoundation/xDrip/assets/92533218/30385636-3d82-4ec4-b5cf-e7b7cbf250ad)
![IMG_20230710_191549](https://github.com/NightscoutFoundation/xDrip/assets/92533218/3d79943f-6f2e-4712-a857-e2e6766338d0)

**Reply by NIkolovafa** [reply]
It seems the connection is good between the phone and the transmitter. It happened with both sensors and after the restart. The only thing that is different is the new transmitter. We have used Dexcom one for over an year and it was working fine. I am really out of ideas. The only thing that I found online with this error is that the transmitter has expired, but it is shown 1 day and it is brand new.

**Reply by jamorham** [reply]
Please enable OB1 debugging as described here: https://github.com/NightscoutFoundation/xDrip/wiki/Enabling-Debugging-for-OB1-collector - try the sensor start again and if it continues to fail please submit the logs using the log upload feature and add some identifier so we can know they are from you.

**Reply by NIkolovafa** [reply]
Thanks @jamorham!I really appreciate all the help.
The logs were sent.

**Reply by Navid200** [reply]
Are you sure you have the correct transmitter in the sensor?
Sometimes we make a mistake and place an old transmitter inside the sensor while xDrip is connected to the new transmitter.
The symptoms you are showing match that classic mistake.  Would you please make sure?

**Reply by NIkolovafa** [reply]
Yes, I am sure. I have double checked today when removed it from the sensor. I read that the error message is related to an expired transmitter. And this is our first transmitter that starts with C. It also shows day 1 on the Status screen. The old one was thrown away. Somehow, I believe the application doesn't read the new transmitter code? Is this possible?

**Reply by jamorham** [reply]
I'm not certain but the data in the logs appears to show that the battery status is one that on G5 we would mark as an error, eg battery very low or some other problem with the transmitter. These status codes are not fully understood. It is a known firmware version so my gut feeling is that there is something actually wrong with this transmitter causing the problems. Nothing else in the interactions or battery reports indicates this but there is a status code I can see which looks bad.

The two next things I would try is first see if it will start with code of 0000 - I strongly suspect it wont and the second thing to try (if you want to) is using the official app . If it starts then you can switch back to xDrip and we can investigate further. I'd be happy to be proven wrong but I doubt that either will work. If using the official app, make sure to set the collection type in xDrip to disabled and to uninstall the official app before resuming with xDrip.

**Reply by NIkolovafa** [reply]
Yep, it will not start with 0000. Unfortunately, the official application is not compatible with mine or my son's phone. I will order a new transmitter, hopefully it starts,and it has nothing to do with any new settings or the fact that they all start with letter now. I was even thinking to do a factory reset, but I doubt this will help in case of fault transmitter.

**Comment by NIkolovafa** [comment]
Thanks all for your help! The issue was in the transmitter, with the new one, all is working as expected. Your time is much appreciated 🤗

**Comment by NIkolovafa** [comment]
Unfortunately, after a day of working with the new transmitter the sensor stopped. It give the same error. Tried with removing the the transmitter and set the sensor again. It will not start. 😔

**Reply by Navid200** [reply]
I wish you had posted when the problem started instead of after removing the transmitter.
Anyway, please always post a screenshot of the system status page.  Hopefully, we will be able to help.  But, without that, it's really not practical.

**Reply by NIkolovafa** [reply]
I should have. There weren't any errors, no changes. I was hoping it will start. My suspension is that the issue  is connected with device now. It is Samsung A13. I tried setting mine as a master, Redmi Note 11, it is warming up, but I doubt it will start. 
![Screenshot_2023-07-15-21-43-22-918_com eveningoutpost dexdrip](https://github.com/NightscoutFoundation/xDrip/assets/92533218/817f932f-66b1-4a4b-a3ed-456ab5a967cd)

**Reply by Navid200** [reply]
You are just wasting your time if you start sensor when there is no connectivity.
Why are you using that version of xDrip?
Please use the latest Nightly: https://github.com/NightscoutFoundation/xDrip/releases/tag/2023.07.13
Please don't start sensor before the connectivity issue is resolved.

**Reply by jamorham** [reply]
The transmitter status showing as `UNKNOWN` is likely very bad and implies permanent transmitter failure.  One transmitter failing I can understand but two seems bad. I don't believe we have had any similar reports elsewhere of the same. This implies the same status code I saw previously on the logs you submitted. We have seen other different problems with these type of transmitters and there could be a quality control issue with manufacture but I don't have enough information to really know for sure.

For safety I would suggest trying companion mode instead in future along with the latest nightly to rule out any software incompatibilities.

**Reply by NIkolovafa** [reply]
How do I found companion mode?I am speaking with another lady and she is reporting issues with two transmitters. There is something going on, probably as you mentioned with the quality control.

**Reply by jonprogers** [reply]
Try these directions for companion mode... 
https://navid200.github.io/xDrip/docs/Follow/CompanionApp.html

**Comment by NIkolovafa** [comment]
Tried with the newest version.
![16894496557256417415204132615539](https://github.com/NightscoutFoundation/xDrip/assets/92533218/4020cda0-bc21-44ca-acd6-f451749bf092)
The connection looks good,but the sensor will not warm up. 😔

**Reply by Navid200** [reply]
You will need to restart.  The error after restart depends on how long ago you inserted the sensor.
https://navid200.github.io/xDrip/docs/Restart-G6-sensor.html

**Reply by Navid200** [reply]
@NIkolovafa Please don't change the xDrip version.  Please don't change the settings.

Please go to the logs.  Scroll up until you get to the logs when the sensor failed.   Please post a screenshot of that screen showing the sensor fail logs.

**Comment by NIkolovafa** [comment]
Unfortunately, I completely whipped the device, started from scratch, downloaded the latest version and now the transmitter won't even connect. I think I gave up for now. My son is not with me to test again for the next couple of days. I can ask him to send the logs, but not sure if this will help.

**Reply by Navid200** [reply]
The last screenshot you have posted, which does not show transmitter status unknown is very promising.
You may only have needed to restart the sensor.

Sensor problems and transmitter connectivity problems are very different.  It is unfortunate when users start the sensor when they should have just waited for the status page to completely populate.

**Comment by NIkolovafa** [comment]
I started the sensor when the transmitter was connected, after following the instructions from the page you provided but it stopped in few minutes. Then, i tried with my phone, it couldn't connect. They I rest my son's phone and tried, won't connect now. Not sure what else I can try. Try with their application but that means to get a new phone. Or try with another transmitter.

**Reply by Navid200** [reply]
I would use the Dexcom app, with the 5-day-old transmitter if I were you to see if the transmitter works or not.

**Reply by Navid200** [reply]
You don't need a new phone.
See this:  

https://docs.google.com/forms/d/e/1FAIpQLScD76G0Y-BlL4tZljaFkjlwuqhT83QlFM5v6ZEfO7gCU98iJQ/viewform?fbzx=2196386787609383750&fbclid=IwAR2aL8Cps1s6W8apUVK-gOqgGpA-McMPJj9Y8emf_P0-_gAsmJs6QwAY-o0

**Reply by NIkolovafa** [reply]
Thanks @Navid200. I was able to download the app on my device and will try on my son's phone Tuesday with the latest transmitter. I sent the previous one to another person, and they will try to run it on Xdrip, different environment. I will let you know the outcome.
